en/peripherals/remote_id.md

```diff
@@ -168,10 +201,48 @@ Some known issues are:
   - PX4 v1.14 does not yet receive `OPEN_DRONE_ID_ARM_STATUS`.
 - `OPEN_DRONE_ID_ARM_STATUS` must be forwarded to the GCS, if present for additional error reporting.
 - [OPEN_DRONE_ID_BASIC_ID](https://mavlink.io/en/messages/common.html#OPEN_DRONE_ID_BASIC_ID) specifies a serial number in an invalid format (not ANSI/CTA-2063 format).
-- The vehicle ID is expected to be tamper resistent.
+- The vehicle ID is expected to be tamper resistant.
 
 [PX4-Autopilot/21647](https://github.com/PX4/PX4-Autopilot/pull/21647) is intended to address the known issues.
 
+## Troubleshooting
+
+### CAN Remote ID Not Working
+
+::: info
+This information was tested with the CAN Cube ID from CubePilot.
+It _should_ also apply to CAN Remote ID modules from other vendors.
+:::
+
+To confirm that the Remote ID is working:
+
+- Check that the Remote_ID node appears on the UAVCAN list.
+
+  Run the following command in the [QGroundControl MAVLink Console](http://localhost:5173/px4_user_guide/en/debug/mavlink_shell.html#qgroundcontrol-mavlink-console):
+
+  ```sh
+  uavcan status
+  ```
+
+  The remote id should appears in the list as shown below (there maybe other nodes in the list too).
+
+  ```plain
+  Online nodes (Node ID, Health, Mode):
+     125 OK         OPERAT
+  ```
+
+- Check that the `OPEN_DRONE_ID_BASIC_ID` and `OPEN_DRONE_ID_LOCATION` messages appear in the QGroundControl [MAVLink Inspector](https://docs.qgroundcontrol.com/master/en/qgc-user-guide/analyze_view/mavlink_inspector.html) (QGC **Analyze Tools > MAVLink Inspector**).
+
+If the above are not true, then the Remote ID itself may need to be configured:
+
+1. Open QGroundControl
+2. Navigate to the [Application settings](https://docs.qgroundcontrol.com/master/en/qgc-user-guide/settings_view/general.html): **Application Settings > General > Miscellaneous**.
+3. Select `Enable Remote ID`.
```

Comment:
Well to be more clear it is in the miscellaneous section of the general tab, but yes there is a checkbox to enable remote ID on QGC which will then enable the Remote ID tab so that data like operator ID, self ID, and Basic ID can be set for the drone and ground station.

Reply:
@Aaron1356 Hmmm. On my QGC daily I have a dedicated tab as shown below. What version are you using?

Either way, it sounds like we send them to this page depending on the version of QGC used.

![image](https://github.com/user-attachments/assets/8a8f0642-0178-4f93-a38a-cfd94ec2841c)

Reply:
I think I'll just note some of the options